jax/experimental/array_serialization/serialization.py

```diff
@@ -63,9 +63,6 @@ class BarrierTimeoutException(Exception):
     "Suggestions for possible fixes:\n"
     "* Check the logs to see if one or more processes failed.\n"
     "* Make sure the training and checkpointing endpoints are close geographically.\n"
-    "* Try setting these environment variables: "
```

Comment:
Let's keep this just in case?

Reply:
it'll be a bit weird if it's already set. should I sniff those out and change the error message if they're already set to something? (fwiw, Apple sets the bytes one to 1024 I think)

Reply:
No, it's fine. let's get this in.

Reply:
reverted

Reply:
Ohh why?

Reply:
i thought that's what you wanted me to do? I can revert the reversion

Reply:
Yeah.. I meant that let's go with your original CL.

Reply:
the reverters have been reverted